flowzone.yml

```diff
@@ -232,6 +232,11 @@ on:
         type: string
         required: false
         default: ""
+      docusaurus_website:
+        description: "Runs a custom command to build the website. The command needs to be in a npm script stored in `npm run deploy-docs`. Skipped if empty or not true."
```

Comment:
This description needs to reflect that it enables docusaurus, and if disabled it will run an npm script instead.